### PR TITLE
Feature 1 is completed

### PR DESCRIPTION
**Pull request by joindevopscloud**
- feature 1 development is completed
- added user sign up
- added login functionality

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,2 +1,6 @@
 ### GIT
-THis is the first commit from main branch
\ No newline at end of file
+THis is the first commit from main branch
+
```
Comment (sivakumarreddym) [inline README.MD:3]: is this line required?

**Review by sivakumarreddym — COMMENTED** [review]
is this line really required? can you chane

File: README.MD (hunk 1 above)
Comment (joindevopscloud) [reply README.MD:3]: yes, that is required

**Review by sivakumarreddym — APPROVED** [review]

**Review by sivakumarreddym — APPROVED** [review]
approving this PR, everything seems fine

**Pull request merged**